Digital Shadows/digital_shadows_searchlight/_meta/fields.yml

```diff
@@ -0,0 +1,44 @@
+event.id:
```

Comment:
It already exists an ECS field with the exact same name: `event.id`.
If you want to parse the Digital Shadow event id, you could store it in a dedicated field such as `digital-shadows.event.id`